kubernetes-client/pom.xml

```
@@ -220,7 +216,8 @@
                         META-INF/MANIFEST.MF,
                         META-INF/maven,
                         META-INF/maven/**/*,
-                        io/fabric8/kubernetes/client/dsl/internal/OperationSupport.*
+                        io/fabric8/kubernetes/client/dsl/internal/OperationSupport.*,
+                        io/fabric8/kubernetes/client/osgi/*.*
```

Ah, right, it's not a regex, so everything under `osgi` package is just:
```suggestion
                        io/fabric8/kubernetes/client/osgi/*
```